Kickstarter-iOS/Library/Nib.swift

```diff
@@ -3,6 +3,7 @@ import UIKit
 public enum Nib: String {
   case BackerDashboardEmptyStateCell
   case BackerDashboardProjectCell
+  case CreatePasswordFooterView
```

Comment:
Do we still need this class if we're going to use `SettingsGroupedFooterView`?

Reply:
Actually I forgot to do the cleanup. Will delete the unused files 👍